### PR TITLE
Nan and scale action wrappers

### PR DESCRIPTION
**Pull request by Kyle-Sang**
Adds scale action and NaN wrappers to the aec_wrappers.py file. Documentation has been updated to include the functions from the new wrappers. Additionally, there are 2 test files (NaN_test.py and scale_action_test.py) as well as their associated dummy environments used in tests (dummy_NaN_env.py and dummy_scale_action_env.py).

### CODE REVIEW

File: supersuit/aec_wrappers.py [hunk 1]
```diff
@@ -548,3 +550,64 @@ def __init__(self, env, lower_bound=-1, upper_bound=1):
 
     def _change_reward_fn(self, rew):
         return max(min(rew, self.upper_bound), self.lower_bound)
+
+class nan_noop_wrapper(BaseWrapper):
+    '''
+    this wrapper expects there to be a no_op_action parameter which
+    is the action to take in cases when nothing should be done.
+    '''
+    def step(self, action, no_op_action, observe = True):
```
Comment (weepingwillowben) [inline supersuit/aec_wrappers.py:559]: `no_op_action` should be an argument to `__init__`, not step.

File: supersuit/aec_wrappers.py [hunk 2]
```diff
@@ -548,3 +550,64 @@ def __init__(self, env, lower_bound=-1, upper_bound=1):
...
+            warnings.warn("[WARNING]: Step received an NaN action {}. Evironment is {}. Taking no operation action.".format(action, self))
+            action = no_op_action
+        return super().step(action)
+
+class nan_zeros_wrapper(BaseWrapper):
+    '''
+    this wrapper warns and executes a zeros action when nothing should be done.
+    Only for Box action spaces.
+    '''
+    def step(self, action, observe=True):
```
Comment (weepingwillowben) [inline supersuit/aec_wrappers.py:570]: `observe=True` as an argument to `step` was removed from the API. So even though the old examples had it, it should not be there in supersuit now. Sorry I didn't mention this earlier

**Review by weepingwillowben — COMMENTED** [review]
There were a couple small issues I noted in the code.

CI Tests are now running on your PR. Try to get the tests to pass. 

If you want to run tests locally, just run

```
pytest test
```

You may have to pip install pytest to get this to work.

**Comment by benblack769** [comment]
Replaced with #94

**Pull request closed without merging**